testing_system/preproccesed_data_generator/preprocessed_data_generator.py

```diff
@@ -0,0 +1,56 @@
+import importlib.machinery
+import os
+import zlib
+from typing import Callable
+
+from testing_system.code_extractor.code_extractor import NtbCodeExtractor
+from testing_system.code_transformers.io_unifier import replace_abs_path_with_relative_path
+from testing_system.code_wrapper.code_wrapper import CodeWrapper
+from testing_system.util.file_util import create_py_file
+
+from testing_system.util.file_util import cwd
+
+class PreprocessedDataGenerator:
+    def __init__(self,
+                 path_to_ntb: str,
+                 kaggle_dir: str):
+        self.path_to_ntb = path_to_ntb
+        self.kaggle_dir = kaggle_dir
+        self.kaggle_solution_dir = os.path.join(kaggle_dir, 'solution')
+
+    def get_preprocessed_data(self):
+        nce = NtbCodeExtractor(self.path_to_ntb)
+        preproc_script_src = nce.get_preprocessing_script()
+        preproc_script_src = replace_abs_path_with_relative_path(preproc_script_src)
```

Comment:
We need to figure something out with paths - currently we have a mess

Reply:
Can you explain what is the problem here with paths?

Reply:
lets discuss it on next meeting